protos/flyteidl/core/interface.proto

```diff
@@ -44,8 +50,14 @@ message Parameter {
     }
 }
 
-// A map of Parameters.
+// A map of Parameters in the form of repeated map entries.
 message ParameterMap {
     // Defines a map of parameter names to parameters.
-    map<string, Parameter> parameters = 1;
+    repeated ParameterMapFieldEntry parameters = 1;
+}
+
+// Map entry of ParameterMap
+message ParameterMapFieldEntry {
+    string key = 1;
+    Parameter value = 2;
```

Comment:
ditto

Reply:
@EngHabu did you want this to be "var"? seems kinda incongruous with `Parameter`